libraries/TGRSIAnalysis/TTigress/TTigress.cxx

```diff
@@ -426,42 +465,50 @@ void TTigress::BuildAddBack(Option_t *opt)	{
     	return;
 
 	addback_hits.clear();
-	addback_hits.push_back(*(this->GetTigressHit(0)));
-	addback_hits.at(0).Add(&(addback_hits.at(0)));
 
-	for(int i = 1; i<this->GetMultiplicity(); i++)   {
-    	bool used = false;
-	    for(int j =0; j<addback_hits.size();j++)    {
-    	    TVector3 res = addback_hits.at(j).GetLastHit() - this->GetTigressHit(i)->GetPosition();
-        	int d_time = abs(addback_hits.at(j).GetTime() - this->GetTigressHit(i)->GetTime());
+	if(this->GetMultiplicity() == 1) addback_hits.push_back(*(this->GetTigressHit(0)));
 
-			int seg1 = std::get<2>(addback_hits.at(j).GetLastPosition());
-			int seg2 = this->GetTigressHit(i)->GetInitialHit();
-		
-			if( (seg1<5 && seg2<5) || (seg1>4 && seg2>4) )	{
-		        if( (res.Mag() < 54) && (d_time < 11) )    {
-    		        used = true;
-        		    addback_hits.at(j).Add(this->GetTigressHit(i));
-            		break;
-	        	}
-			}
-			else if( (seg1<5 && seg2>4) || (seg1>4 && seg2<5) )	{
-		        if( (res.Mag() < 105) && (d_time < 11) )    {
-    		        used = true;
-        		    addback_hits.at(j).Add(this->GetTigressHit(i));
-            		break;
-	        	}
-			}
+	else{
 
+		addback_hits.push_back(*(this->GetTigressHit(0)));
+		addback_hits.at(0).Add(&(addback_hits.at(0)));
```

Comment:
Does this line add the first interaction to itself? I realize it's the same as the previous version, but I'm curious all the same.